sources/engine/Stride.Assets.Models/ModelAssetImporter.cs

```diff
@@ -291,7 +315,8 @@ private static void ImportTextures(IEnumerable<string> textureDependencies, List
 
             foreach (var textureFullPath in textureDependencies.Distinct(x => x))
             {
-                var texturePath = new UFile(textureFullPath);
+                if (!System.IO.File.Exists(textureFullPath)) { continue; }
+                var texturePath = new UFile(textureFullPath);      
```

Comment:
Might want to log missing texture as warning ?

Reply:
fixed